### DISCUSSION TITLE
Citing Sources on Assignment 1

### DISCUSSION THREAD

**Discussion by sabarrag** [CS342/.github#13] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

Hi! How would we go about citing external sources we used for inspiration on our Assignment 1?

### Reproduction

N/A

### Expected behavior

N/A

### Additional context

N/A

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Hi @sabarrag,

Please add any sources that you have used in the PR description, e.g., you can use the "Additional Context" section in the PR, as well as code comments within the source code in your Swift file.
A link to the source + a bit of context how/why you have used that should be fine.

**Reply by sabarrag** [reply]
Okay sounds good! Thank you!
